content/pt-br/docs/tasks/configure-pod-container/extended-resource.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Atribuindo Recursos Estendidos a um Contêiner
+content_type: task
+weight: 40
+update_date: 2022-07-17
+origin_version: 1.24
+contributors: DonatoHorn
```

Comment:
Git tracks contributions. No need to add this field.

Reply:
Hi, I'm new contributor on pt-br docs, so my difficults are identify pages (original version) than where used base-line for internacionalizations, in pt-br docs we have pages from v1.18, 1,19, 1.20, 1.21, 1.22, 1.23, 1.24, how can I guess what pages need update. I'ts nice to do this, make a diff in the english docs, so the pages that need update will be shown. But in pt-br we don't have versioning, so if I have a origin_version  in the docs I can use a grep to show what a version used in, and assign the contributor to ask if interested in update the docs they translated.

Reply:
I would ask folks in the Slack channel for localizations for help on doing this. We have tools that you can use, and I hope someone will be happy to show you.

Reply:
This issue with outdated localization was previously discussed in issue #29649. Would having automation like this require some kind of RFC to be implemented?